.gitignore

```diff
@@ -1,68 +1,60 @@
 # Don't add any custom patterns for platform-specific files like .DS_Store
 # or generated by custom development tools like .vscode or .idea that don't
 # affect other developers. Instead add those custom patterns to your global
-# .gitignore file.
+# gitignore or to your local .git/info/exclude.
 
+# Logs, local, temp files
 *.diff
 *.log
+mixxx.log*
```

Comment:
mixxx.log is not in the source tree isn't it?

Reply:
Whenever I execute the tests I get ten mixxxx-log files in the root directory ^^

Reply:
yeah, me too.
Should we rather prevent that in the first places and create logs in the build/test dir?

Reply:
we should, but I think I tried that and it was a little more complex...

Reply:
I think it's related to https://github.com/mixxxdj/mixxx/pull/2938